ui/Screen/TransactionDetails.svelte

```diff
@@ -48,7 +48,10 @@
         text={transaction.statusText(tx.state)}
         variant={transaction.iconState(tx.state)}
         style="margin-bottom: 32px; margin-top: 96px;" />
-      <Transaction transaction={tx} {payer} />
+      <Transaction
+        transaction={tx}
+        {payer}
+        transactionCosts={session.transactionCosts} />
```

Comment:
Indeed. We'll have the same problem with currency conversion rates if we don't explicitly cache the numbers on disk.

Here I assume that the transaction costs won't change very often. Since they're hardcoded in the registry, we'd need an application update to change them, in this case we would have to either do a migration of the cached transactions or drop them altogether.

Would this naive version be okay for now @cloudhead?

Reply:
Yeah I mean the current suggested fee and the actual payed fee are different things. I would assume that the later is available from looking at the transaction data that we store in the proxy.

So when submitting a new transaction, we see the suggested fee, and when looking at a past transactions, we should see the actual fee used.

They happen to be the same for now, but this will break eventually, so I'd open an issue and try to fix it before launch :+1:

Reply:
> They happen to be the same for now, but this will break eventually, so I'd open an issue and try to fix it before launch 👍

The current implementation reads the minimum fee from the backend and the UI passes it back to proxy via the submitted transaction, so technically there's nothing in the way to let the user select a fee in the UI.

Reply:
Yeah that's fine, but the tx history will still break eventually, when the fees used change.

Reply:
Added an issue: https://github.com/radicle-dev/radicle-upstream/issues/473.